app/controllers/split_checkout_controller.rb

```diff
@@ -0,0 +1,44 @@
+class SplitCheckoutController < CheckoutController
```

Comment:
I think we'll be better off if we avoid inheritance here and create a fresh controller that's not subclassing the old one.